src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,24 @@
-export const TodoInfo = () => {};
+import { UserInfo } from '../UserInfo';
+import { Todos } from '../../types/TodosProps';
+
+type TodoInfoProps = {
+  todo: Todos
+};
+
+export const TodoInfo: React.FC<TodoInfoProps> = ({ todo }) => {
+  return (
+    <article
+      className={`TodoInfo ${todo.completed && 'TodoInfo--completed'}`}
```

Comment:
<img width="198" alt="image" src="https://github.com/mate-academy/react_add-todo-form/assets/70220667/f3c11356-6e99-4081-99cc-3cc97bf6294b">
we usually use classnames package for conditional classes